pkg/doc/vc/profile/profile.go

```diff
@@ -40,6 +40,7 @@ type DataProfile struct {
 	Creator                 string                             `json:"creator"`
 	Created                 *time.Time                         `json:"created"`
 	DIDPrivateKey           string                             `json:"didPrivateKey"`
+	DisableVCStatus         bool                               `json:"disableVCStatus"`
```

Comment:
note: I think it would be better to think about this as the default behaviour for this profile rather than as "Disable"